plugin-management-cli/src/main/java/io/jenkins/tools/pluginmanager/cli/Main.java

```diff
@@ -20,9 +20,15 @@ public static void main(String[] args) throws IOException {
             throw new IOException("Failed to read command-line arguments", e);
         }
 
-        Config cfg = options.setup();
-        PluginManager pm = new PluginManager(cfg);
-        pm.start();
+        if (options.isShowVersion()) {
+            options.showVersion();
+            System.exit(0);
```

Comment:
For java, you should probably use
```suggestion
            return;
```